### ISSUE TITLE
CTk 5.1.2: JSON Property vs widget name inconsistencies

### ISSUE THREAD

**Issue by avalon60** [TomSchimansky/CustomTkinter#1538]
For some widgets, the theme JSON (widget names) are inconsistent with the name of the actual widget:
```

Widget  Name             Property Name
===========              ============
CTkCheckBox              CTkCheckbox
CTkRadioButton           CTkRadiobutton
CTkTextBox               CTkTextbox
```

This has a knock-on effect, when coding around widgets and themes. For example, it  requires this otherwise unnecessary "if ladder":

```
        for property, parameters in widget_parameter.items():
            # This "if ladder" is necessary, because there are several naming
            # inconsistencies (at least in CTk 5.1.2), between widget names
            # and the name used in the JSON.
            if widget_type == 'CTkCheckBox':
                json_widget_type = 'CTkCheckbox'
            elif widget_type == 'CTkRadioButton':
                json_widget_type = 'CTkRadiobutton'
            elif widget_type == 'CTkTextBox':
                json_widget_type = 'CTkTextbox'
            else:
                json_widget_type = widget_type
            current_value = int(self._theme_json_data[json_widget_type][property])
```

Thanks,

Clive

**Comment by TomSchimansky** [comment]
This is what I added now in the ThemeManager:
```python
        # fix name inconsistencies
        if "CTkCheckbox" in cls.theme.keys():
            cls.theme["CTkCheckBox"] = cls.theme.pop("CTkCheckbox")
        if "CTkRadiobutton" in cls.theme.keys():
            cls.theme["CTkRadioButton"] = cls.theme.pop("CTkRadiobutton")
```
The "CTkTextbox" name is indeed correct, only "CTkCheckbox" and "CTkRadiobutton" were named differently than the classes.

This attempt will however brake the program for people who manipulate the theme at runtime but that's how it is. Im not planning on a new major release in the next time so I will just push this with a minor release.

**Comment by TomSchimansky** [comment]
fixed with version 5.2
